terraform/services/api-rds/main.tf

```diff
@@ -2,47 +2,130 @@ locals {
   db_name = {
     ab2d = {
       dev  = "ab2d-dev"
-      test = "ab2d-east-prod-test"
-      sbx  = "ab2d-sbx-sandbox"
+      test = "ab2d-east-impl"
       prod = "ab2d-east-prod"
-    }
-    bcda = {
-      dev  = "${var.app}-${var.env}"
-      test = "${var.app}-${var.env}"
-      sbx  = "${var.app}-${var.env}"
-      prod = "${var.app}-${var.env}"
-    }
-    dpc = {
-      dev  = "${var.app}-${var.env}"
-      test = "${var.app}-${var.env}"
-      sbx  = "${var.app}-${var.env}"
-      prod = "${var.app}-${var.env}"
-    }
```

Comment:
Could we leave something in here for bcda and dpc so the plan checks pass?

Reply:
Added a variation on this in the latest push - will look at updating the locals to set this up for success in the other environments on testing